pkg/northbound/admin/admin.go

```diff
@@ -76,13 +78,23 @@ func (s Server) ListRegisteredModels(req *proto.ListModelsRequest, stream proto.
 			schemaEntries = append(schemaEntries, &schemaEntry)
 		}
 
+		roPaths := make([]string, 0)
+		if req.Verbose {
+			var ok bool
```

Comment:
why do you need to define the ok variable ?

Reply:
because I can't use ":=" in the next line because roPaths exists already

Reply:
because I can't use ":=" in the next line

Reply:
you can still do := if there is at least one new variable to the left of the assignment.

Reply:
there are bigger issues than this

Reply:
doesn't work in this case